integrated_channels/moodle/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+"""
+The Moodle Integrated Channel package.
+"""
+
+__version__ = "0.1.0"
+
+default_app_config = ("integrated_channels.moodle.apps."  # pylint: disable=invalid-name
```

Comment:
hmm the name looks valid, do we still need that disable override for pylint?

Reply:
just tested and yeah no it's still invalid. It's because `default_app_config` doesn't match the constant format (ie something like `DEFAULT_APP_CONFIG`) Why every integrated channel is lowercase with this idk. I just assumed django magic.